cfg/TerarangerHubEvo.cfg

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+from dynamic_reconfigure.parameter_generator_catkin import *
+PACKAGE = "teraranger_evo_cfg"
+
+gen = ParameterGenerator()
+
+mode_enum = gen.enum([gen.const("Binary", int_t, 0, "Enable binary mode"),
+                     gen.const("Text", int_t, 1, "Enable text mode")],
+                     "Set the working mode")
+
+mode_range_enum = gen.enum([gen.const("Long_range", int_t, 0, "Enable Long_range mode"),
```

Comment:
We should see with Tech and commercial teams about those two modes. From what I've heard the sensor will work only in a single mode